packages/react/src/forms/FormControl/FormControl.test.tsx

```diff
@@ -88,6 +89,42 @@ describe('FormControl', () => {
     expect(rootEl.getAttribute('id')).toBe(`FormControl--${mockFormControlId}`)
   })
 
+  it('can forward a name prop to text inputs', async () => {
+    const mockName = 'input-name'
+    const {getByRole} = render(
+      <FormControl>
+        <FormControl.Label>{mockFormControlLabel}</FormControl.Label>
+        <TextInput name={mockName} />
+      </FormControl>,
+    )
+
+    const inputEl = getByRole('textbox')
+
+    const inputNameValue = inputEl.getAttribute('name')
+
+    expect(inputNameValue).toEqual(mockName)
```

Comment:
Just fyi Testing Library exposes a [`toHaveAttribute` matcher](https://github.com/testing-library/jest-dom?tab=readme-ov-file#tohaveattribute) which can wrap all this up into a single expect

```ts
expect(getByRole('textbox')).toHaveAttribute('name', mockName)
```

Reply:
Ah, or even just 

```ts
expect(getByRole('textbox', {name: mockName})).toBeInTheDocument()
```

Reply:
👍 thanks Josh, gonna keep these as-is for now to avoid rewriting all the other test cases that use the native methods. Don't see it as a huge issue right now, as it should achieve the same result and not affect results AFAIK.